apps/client/src/routes/dashboards/create/components/create-dashboard-form-actions/cancel-create-dashboard-button.tsx

```diff
@@ -0,0 +1,23 @@
+import Button from '@cloudscape-design/components/button';
+import { FormattedMessage } from 'react-intl';
+
+import { DASHBOARDS_HREF } from '~/constants';
+import { useApplication } from '~/hooks/application/use-application';
+
+/** Button used to cancel dashboard creation and return to dashboards page */
+export function CancelCreateDashboardButton() {
```

Comment:
You could consolidate the `CancelCreateDashboardButton` & `ConfirmCreateDashboardButton` into a single reusable button component. Ex: 

```import Button from '@cloudscape-design/components/button';
import { FormattedMessage } from 'react-intl';
import { DASHBOARDS_HREF } from '~/constants';
import { useApplication } from '~/hooks/application/use-application';

export interface DashboardButtonProps {
  defaultMessage: string;
  description: string;
  isLoading?: boolean;
}

export const DashboardButton = (props: DashboardButtonProps) => {
  const { defaultMessage, description, isLoading } = props;
  const { navigate } = useApplication();

  const handleClickCancel = () => {
    navigate(DASHBOARDS_HREF);
  }

  const buttonAttributes = isLoading 
    ? {
      variant: "primary",
      loading: {isLoading},
      formAction: "submit",
    }
    : {
      formAction: "none", 
      onClick: {handleClickCancel}
    };

  return (
    <Button {...buttonAttributes} >
      <FormattedMessage
        defaultMessage={defaultMessage}
        description={description}
      />
    </Button>
  );
}`